mkroosevelt.js

```diff
@@ -45,7 +52,19 @@ else if (cmd === '-v' || cmd === '--v' || cmd === '-version' || cmd === '--versi
 }
 else if (cmd) {
   try {
-    createSampleApp();
+    if (cmd === '.') {
+      currentDirectory = true;
+      stdio.question('Installing in the current directory will DELETE EVERYTHING in the current directory. Are you sure you want to continue (y/n)', function (err, decision) {
```

Comment:
Let's reword this a bit: "This will DELETE EVERYTHING in this directory. Are you sure you want to continue (y/N)"